src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -1,4 +1,5 @@
 import { navigateTo, internalFunctions as navigationManagerInternalFunctions } from './Services/NavigationManager';
+import { protectedBrowserStorage } from './Services/ProtectedBrowserStorage';
```

Comment:
This would have to carry it's own js files since it's no longer tied to the framework.